changelogs/fragments/openssl_privatekey-generate.yaml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - openssl_privatekey can't create files which only have read permissions #48656(https://github.com/ansible/ansible/issues/48656)
```

This is interpreted as YAML, whence `#` and everything after it will be treated as a comment and will be removed. You can work around that by quoting it:
```suggestion
  - "openssl_privatekey can't create files which only have read permissions #48656(https://github.com/ansible/ansible/issues/48656)"
```
Anyway, I would not mention `#48656`, but simply leave the link (in brakets).

---

I will fix the points as you pointed out!